sewer/dns_providers/auroradns.py

```diff
@@ -8,7 +8,7 @@
     from libcloud.dns.types import Provider, RecordType
     import tldextract
 except ImportError:
-    aurora_dependencies = True
+    aurora_dependencies = False
```

Comment:
thanks. 
I do not know how I missed this

Reply:
Thanks to you for your package :)